package PS;
import "proto/neural_network.proto";
import "linear_method/linear_method.proto";
import "graph_partition/parsa.proto";

message AppConfig {
  optional string app_name = 1;
  repeated string parameter_name = 2;
  optional LM.Config linear_method = 3;
  optional NN.Config neural_network = 4;
  optional ParsaConf parsa = 5;
}
